### ISSUE TITLE
Rescoring and Constraints in multimer modeling.

### ISSUE THREAD

**Issue by entropybit** [deepmind/alphafold#586]
Hi,

since I am working on complexes I was very happy about the multimer modeling.
However, if one has available distance constraints from the lab saying which residues interact with 
each other, it would be highly useful if those constraints could be considered somehow.
With a normal docking approach I would just generate a big ensemble and select the models based on my constraints from the ensemble. Of course that leaves me with many models to choose from instead of the multi models produced
by AF2. I was also thinking if I could just rescore those models by using them as AF2 templates but that would still mean a few hundred to thousand AF2 runs depending on the protein complex ^^'.
And I am also not sure if that really is a "rescoring" of that complex with AF2. 
So in summary, considering constraints would be really great, and also some kind of rescoring mechanism to
gauge the quality of an arbitrary input structure with the trained model.

Regards and Thanks in advance,
Benjamin Mayer,
TUD, Germany
